bin/merge_boa.r

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env Rscript
+
+args = commandArgs(trailingOnly=TRUE)
+
+
+if (length(args) < 3) {
+    stop("\nthis program needs at least 3 inputs\n1: output filename\n2-*: input files", call.=FALSE)
+}
+
+fout <- args[1]
+finp <- args[2:length(args)]
+nf <- length(finp)
+
+require(raster)
+
+
+img <- brick(finp[1])
+nc <- ncell(img)
+nb <- nbands(img)
+
+
+sum <- matrix(0, nc, nb)
+num <- matrix(0, nc, nb)
+
+for (i in 1:nf){
+
+    data <- brick(finp[i])[]
+
+    num <- num + !is.na(data)
+
+    data[is.na(data)] <- 0
+    sum <- sum + data
+
+}
```

Comment:
This highly depends on the type, size and overlap of the pipeline's input data. It may become >100 for some extreme cases, but for our currently used data its usually between 5 and 20. The merge scripts are mostly untouched from the previous (not nf-core) installation of this pipeline. I will rework them and also include the other changes you suggested.